### DISCUSSION TITLE
Change the model's name in chat

### DISCUSSION THREAD

**Discussion by JuanjoJG-WATA** [open-webui/open-webui#1563] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
I think it would be nice to be able to change the name of the model in chat. In my case it shows just as "Gpt-3.5-Turbo", but I want to change it to a more personalized one.

**Describe the solution you'd like**
A way to change this easily, maybe just changing a svelte file.

**Describe alternatives you've considered**
I've tried looking for the model's name in variables, but I got nothing

**Additional context**
**_Before_**
![image](https://github.com/open-webui/open-webui/assets/165764759/2d66eb9f-2708-46c0-b108-ac31f8ed18e8)

**_After_**
![Sin título](https://github.com/open-webui/open-webui/assets/165764759/72650139-1bf5-4c6a-87ee-a987e191fc0f)

**Comment by JuanjoJG-WATA** [comment, marked as answer]
Found it.
You just need to change the .yaml file for the model. As I'm using an Azure OpenAI API key, i just had to change the model's name there to also change in the chat and in the Select Model Menu.

**Reply by zubu007** [reply]
what do i have to change in the .yaml file? Is there any model name variable?

**Comment by Hugobox** [comment]
Hi! My model is OpenGVLab/InternVL-Chat-V1-5-AWQ but there is no yaml file included with that model. Using LMdeploy to serve it. Havn't found a way to change the model name yet.

**Reply by justinh-rahb** [reply]
![341838802-cce89473-a612-45a9-9e9d-822923985d88](https://github.com/open-webui/open-webui/assets/52832301/a313486f-57d1-4631-b262-f0d8eba3e736)

**Reply by Hugobox** [reply]
Thanks! This works beautifully!
